pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1392,6 +1392,7 @@ func TestAlgorithmProviderCompatibility(t *testing.T) {
 		},
 		"PostFilterPlugin": {
 			{Name: "InterPodAffinity"},
+			{Name: "DefaultPodTopologySpread"},
```

Comment:
all compatibility tests need updating, adding the post filter to the legacy registry requires it.